tests/phpunit/admin/class-wc-stripe-whats-new-modal-test.php

```
@@ -0,0 +1,377 @@
+	 */
+	private function set_up_ajax_context(): void {
+		add_filter( 'wp_doing_ajax', '__return_true' );
+		add_filter(
+			'wp_die_ajax_handler',
+			static function () {
+				return static function ( $message ) {
+					throw new WPDieException( is_string( $message ) ? $message : '' );
+				};
+			}
+		);
+	}
```

PHPMD (2.15.0)

[error] The method set_up_ajax_context is not named in camelCase. (undefined)

(CamelCaseMethodName)